src/components/Wizards.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import ItemList from './ItemList';
+import wizard1 from './../assets/image/zaklecia/wizards1.png';
+import wizard2 from './../assets/image/zaklecia/wiazard2.png';
+
+const Wizards = () => {
+  const [items, setItems] = useState([]);
+
+  const availableItems = [
+    {
+      id: 1,
+      name: 'Fireball',
+      image: wizard1,
+      stats: { strength: 0, power: 5 },
+    },
+    {
+      id: 2,
+      name: 'Ice Shield',
+      image: wizard2,
+      stats: { strength: 2, power: 2 },
+    },
+  ];
+
+  const handleAddItem = (item) => {
+    setItems((prevItems) => [...prevItems, item]);
+  };
```

Comment:
Nie sprawdzamy czy już podana nazwa nie istnieje, lub to samo id? Nie chcesz żadnych warunków ustalać?